client/android/app/src/main/java/com/snapscreen/MainApplication.java

```diff
@@ -24,6 +25,7 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
               new MainReactPackage(),
+            new ReactNativeConfigPackage(),
```

Comment:
I've added spotless.  It is using Google's code formatting rules.  I chose the ruleset out of a hat.